### DISCUSSION TITLE
Is there a way to make the lines generate along the edges of ALPHA？

### DISCUSSION THREAD

**Discussion by mrprotonff** [bnpr/Malt#461] [Q&A] [answered]
Is there a way to make the lines generate along the edges of ALPHA？
![image](https://user-images.githubusercontent.com/121292082/209324129-0fdd2649-cbad-465b-89ee-c0e3443b01b4.png)
![image](https://user-images.githubusercontent.com/121292082/209324212-d2e70ffc-a37c-4df8-889f-5ee660193645.png)

**Comment by mrprotonff** [comment]
The picture below is a transparent PNG with edge lines drawn

**Comment by HSDSZ** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/36077492/209356704-8e76ec19-4084-4cd9-91e2-b3e6f5d92372.png)

**Reply by mrprotonff** [reply]
Thanks a lot！！
